release-notes/opensearch-dashboards.release-notes-2.5.0.md

```diff
@@ -0,0 +1,70 @@
+## Version 2.5.0 Release Notes
+
+### 🛡 Security
+
+- Add Security-CVEs fixes guidelines ([#2674](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2674))
+- Bump decode-uri-component from 0.2.0 to 0.2.2 ([3009](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3009))
+- [CVE-2022-25912] Bumps simple-git from 3.4.0 to 3.15.0 ([#3036](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3036))
```

Comment:
I see some inconsistencies in the title. For example some use `Bump` and some use `Bumps`, could we help developers to polish these titles?

Reply:
For commit messages, we should prefer the command version ("bump", not "bumps")